plugins/fpc/fu-fpc-device.c

```diff
@@ -59,6 +60,14 @@
 
 #define FU_FPC_DEVICE_FLAG_RTS_DEVICE (1 << 2)
 
+/**
+ * FU_FPC_DEVICE_FLAG_LENFY_DEVICE:
+ *
+ * Device is a LENFY MOH device
```

Comment:
can you expand out what LENFY means?

Reply:
Lenfy is our inner project name.
I'll change it.